### ISSUE TITLE
Flaky test Microsoft.AspNetCore.Components.E2ETest.Tests.DownloadAnchorTest.DownloadFileFromAnchor

### ISSUE THREAD

**Issue by TanayParikh** [dotnet/aspnetcore#29739]
```
   at Microsoft.AspNetCore.Components.E2ETest.Tests.DownloadAnchorTest.DownloadFileFromAnchor() in /_/src/Components/test/E2ETest/Tests/DownloadAnchorTest.cs:line 48
```

https://dev.azure.com/dnceng/public/_build/results?buildId=970420&view=ms.vss-test-web.build-test-results-tab&runId=30490800&resultId=100005&paneView=debug

Extension of tests quarantined in https://github.com/dotnet/aspnetcore/pull/29652 / https://github.com/dotnet/aspnetcore/issues/29698.

**Comment by captainsafia** [comment]
Hm -- this test has been passing since it was quarantined.

I noticed that this particular failure occurred when the change to run all quarantined tests as CI runs was reverted so I wonder if that had something to do with it.

Recommend we see if this can be unquarantined next week and then verifying if failures continue once it is running in the regular CI.

Side-note: if we have a hypothesis that the environment the tests run on is affecting their flakiness then we should probably sort that out separately.

**Comment by TanayParikh** [comment]
> Hm -- this test has been passing since it was quarantined.

Where are you seeing this? I'd expect to see it in the [quarantine tests dashboard](https://dev.azure.com/dnceng/public/_test/analytics?definitionId=331&contextType=build) but I'm not finding it.

**Comment by captainsafia** [comment]
Does [this](https://dev.azure.com/dnceng/public/_test/analytics?definitionId=869&contextType=build) work for you?

Also, looks like it got flaky again towards the end of last month. The bigger red bar is due to some issues with Selenium but the other incidents are repros of the flakiness. Bummer. :/

![Screen Shot 2021-03-03 at 10 51 17 AM](https://user-images.githubusercontent.com/1857993/109856517-6cb32c00-7c0e-11eb-809a-50e6cb1637e4.png)

**Comment by TanayParikh** [comment]
Ah got it, thanks! Yeah, looks like the flakiness has returned :/

**Comment by captainsafia** [comment]
Put in a fix for this. Not sure if we are supposed to be closing these now or once 30 days have passed but I'll close and we can re-open if need be.

**Comment by HaoK** [comment]
Well we made this test consistent, but probably not in the way we want, its been full red since 3/11 which I think maps to when you made this change 😢

**Comment by captainsafia** [comment]
Boo. Maybe we can we move this test to the new Playwright E2E tests project and use the [Download APIs](https://playwrightsharp.dev/api/PlaywrightSharp.Download.html) from Playwright?

**Comment by HaoK** [comment]
Sure sounds like a good candidate for early conversion once its ready

**Comment by captainsafia** [comment]
Moving this to Next sprint planning pending the Playwright move so we can use the APIs mentioned above.

**Comment by captainsafia** [comment]
This is "Done" but was merged into the Playwright PR since the fix requires using some APIs that are only available in Playwright. Will close once that PR is merged into main.

**Comment by BrennanConroy** [comment]
> merged into the Playwright PR since the fix requires using some APIs that are only available in Playwright

What PR is that? If it was merged, this test still has a 0% pass rate.

**Comment by captainsafia** [comment]
@BrennanConroy It's https://github.com/dotnet/aspnetcore/pull/32428

**Comment by captainsafia** [comment]
The PR with the Playwright tests has now been merged.
